x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/execution_command_shell_started_by_svchost.json

```diff
@@ -45,5 +45,5 @@
   "timeline_title": "Comprehensive Process Timeline",
   "timestamp_override": "event.ingested",
   "type": "eql",
-  "version": 10
+  "version": 12
```

Comment:
This is ok - from the manually reconciled version bumps elastic/detection-rules/pull/1950